src/main/java/gregtech/integration/jei/multiblock/MultiblockInfoPage.java

```diff
@@ -26,4 +38,51 @@ public float getDefaultZoom() {
                 .map(I18n::format)
                 .collect(Collectors.toList());
     }
+
+    /**
+     * Gets the Map containing all tooltips for MultiblockAbilities. Will generate the map if it does not exist
+     * @return - The Map containing Tooltips and Formatting for specific MultiblockAbilities
+     */
+    public Map<ItemStack, List<Tuple<String, TextFormatting>>> getAbilityTooltipMap() {
+        // Generate on first use
+        if (this.abilityTooltips.size() == 0) {
+            generateAbilityTooltips();
+        }
+        return this.abilityTooltips;
+    }
+
+    /**
+     * Contains the default tooltips that will be applied to all MultiblockAbilities
+     */
+    protected void generateAbilityTooltips() {
+
+        for(int i  = 0; i < GTValues.V.length; i++) {
+            addAbilityTooltip(MetaTileEntities.ITEM_EXPORT_BUS[i].getStackForm(), new Tuple<>("gregtech.multiblock.preview.any_hatch", TextFormatting.GREEN));
```

Comment:
These tuple objects (or ITextComponent if you change it) are immutable.
No need to create multiple copies, use a static field.